pages/verify.md

```diff
@@ -1,5 +0,0 @@
----
-title: Verify
-entryname: verify
-layout: page-react.html
----
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-f43bc125fbb8888abf94eaf76e7cb4ebR5.